client/src/components/PublicPages/PublicPages.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Redirect, Route, Switch } from 'react-router-dom'
+
+import AuditPlanner from './AuditPlanner'
+import { Wrapper } from '../Atoms/Wrapper'
+
+const PublicPages: React.FC = () => {
+  return (
+    <Wrapper>
+      <Switch>
+        <Route exact path="/public/audit-planner">
```

Comment:
I'm just now noticing in the mocks a `/plan?...` URL

<img width="345" alt="plan" src="https://user-images.githubusercontent.com/12616928/188742095-2f68f8f7-a67f-48ba-9388-8671c39a4ff6.png">

More than happy to tweak the routing structure if we don't want `/public/` in the URL!

Reply:
Since this is going to be a URL that's typed out and passed around a lot, I want to optimize for that case. That's the motivation behind `arlo.voting.works/plan`. Though now I'm wondering if `arlo.voting.works/planner` might be a bit more clear. Either way, yeah, let's remove `/public/`.